gateways/clients/elastic_search_client.py

```diff
@@ -19,7 +19,10 @@ def __init__(self, client: Optional[Elasticsearch]) -> None:
                 basic_auth=(ELASTIC_USER, ELASTIC_PASSWORD)
             )
 
-        self.utils = ElasticSearchUtils()
+        self.utils = ElasticSearchUtils(elastic_index=elastic_index)
+
+    def run(self, payload: dict) -> dict:
+        return dict(self.client.search(**payload))
 
     def make_query(self, search_text: str, sort_by: str, order: str, search_after: List[str]) -> dict:
```

Comment:
You weren't able to use this method because it's too specific for the other case we had before to list tokens, right?

Shouldn't we extract this logic of building the query from this class, to have the old case build its query outside as well?

Or is there any change this query will be reusable in other situations?

Reply:
I left it this way because every use case that has an index that has searchable fields will be able to reuse this query, even though we are not using it on the token balances query